### PR TITLE
Add new entry to Resource.ApiGroup for Infra features

### PR DESCRIPTION
**Pull request by dbasunag**
##### Short description:
Based on review comment in a cnv-tests PR adding an entry of operators.openshift.io/infrastructure-features to Resource.ApiGroup
##### More details:
As suggested by @rnetser in a cnv-tests PR, adding this entry
##### What this PR does / why we need it:
This would be used in tests

### CODE REVIEW

File: ocp_resources/resource.py [hunk 1]
```diff
@@ -319,6 +319,7 @@ class ApiGroup:
         UPLOAD_CDI_KUBEVIRT_IO = "upload.cdi.kubevirt.io"
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        INFRA_FEATURES = "operators.openshift.io/infrastructure-features"
```
Comment (myakove) [inline ocp_resources/resource.py:322]: this is not match our code, you should add only "operators.openshift.io" here
name should end with _IO and should be placed in the right line (a, b, c ....)

File: ocp_resources/resource.py [hunk 2]
```diff
@@ -319,6 +319,7 @@ class ApiGroup:
         UPLOAD_CDI_KUBEVIRT_IO = "upload.cdi.kubevirt.io"
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        INFRA_FEATURES_IO = "operators.openshift.io"
```
Comment (myakove) [inline ocp_resources/resource.py:322]: please move it to the right place (ordered by abc)

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/resource.py [hunk 3]
```diff
@@ -279,6 +279,7 @@ class ApiGroup:
         HCO_KUBEVIRT_IO = "hco.kubevirt.io"
         HOSTPATHPROVISIONER_KUBEVIRT_IO = "hostpathprovisioner.kubevirt.io"
         IMAGE_OPENSHIFT_IO = "image.openshift.io"
+        INFRA_FEATURES_IO = "operators.openshift.io"
```
[the remark this answers is not in the document]
Comment (dbasunag) [reply ocp_resources/resource.py]: Done

File: ocp_resources/resource.py [hunk 4]
```diff
@@ -297,6 +297,7 @@ class ApiGroup:
         NMSTATE_IO = "nmstate.io"
         NODEMAINTENANCE_KUBEVIRT_IO = "nodemaintenance.kubevirt.io"
         OPERATOR_OPENSHIFT_IO = "operator.openshift.io"
+        OPERATORS_OPENSHIFT_IO = "operators.openshift.io"
```
Comment (myakove) [inline ocp_resources/resource.py:300]: Move it after `OPERATORS_COREOS_COM` please.

**Pull request merged**